src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -225,6 +227,29 @@ internal sealed partial class SyntaxTreeIndex
             }
         }
 
+        private static bool IsGlobalSuppressMessageAttribute(ISyntaxFactsService syntaxFacts, SyntaxNode node)
+        {
+            if (!syntaxFacts.IsGlobalAttribute(node))
+                return false;
+
+            var name = syntaxFacts.GetNameOfAttribute(node);
+            if (syntaxFacts.IsQualifiedName(name))
+            {
+                syntaxFacts.GetPartsOfQualifiedName(name, out _, out _, out var right);
+                name = right;
+            }
+
+            if (!syntaxFacts.IsIdentifierName(name))
```

Comment:
is this even possible with `GetNameOfAttribute`?

Reply:
yes.  `[A.B<int>()]`